src/test/java/com/revature/services/PurchaseServiceTest.java

```diff
@@ -93,6 +95,19 @@ void testFindByOwner() {
 		assertTrue(actual.containsAll(expected));
 		verify(this.mockPurchaseRepo, times(1)).findByOwnerUser(this.dummyUser);
 	}
+	
+	@Test
+	void testFindByOwner_Failure_UserNotFound() {
+		int id = 2;
+		given(this.uServ.findById(id)).willReturn(Optional.empty());
+		
+		try {
+			this.purchaseServ.findByOwner(id);
+			fail("Expected exception to be thrown");
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `UserNotFoundException`